### PR TITLE
[12.0][MIG] base_gs1_barcode

### PR DESCRIPTION
**Pull request by JoanSForgeFlow**
Standard base_gs1_barcode migration to v12.

cc ~ @ForgeFlow @LoisRForgeFlow

### CODE REVIEW

File: base_gs1_barcode/__manifest__.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 {
     'name': 'GS1 Barcode API',
     'summary': 'Decoding API for GS1-128 (aka UCC/EAN-128) and GS1-Datamatrix',
-    'version': '11.0.1.0.1',
+    'version': '12.0.1.0.1',
```
Comment (LoisRForgeFlow) [inline base_gs1_barcode/__manifest__.py]: As per https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0, it should be `12.0.1.0.0`

**Review by LoisRForgeFlow — APPROVED** [review]
Seems a pretty straight forward migration, no much diff to review, LGTM :+1:

File: base_gs1_barcode/__manifest__.py (hunk 1 above)
Comment (JoanSForgeFlow) [reply base_gs1_barcode/__manifest__.py]: Done! Thanks for your review

File: base_gs1_barcode/README.rst [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+This module is part of the `OCA/stock-logistics-barcode <https://github.com/OCA/stock-logistics-barcode/tree/11.0/base_gs1_barcode>`_ project on GitHub.
+
+You are welcome to contribute. To learn how please visit https://odoo-community.org/page/Contribute.
```
Comment (sergio-teruel) [inline base_gs1_barcode/README.rst:122]: You must be generate this file again with oca-readme-gen script...

**Review by sergio-teruel — APPROVED** [review]
Only a comment

**Comment by pedrobaeza** [comment]
As the README.rst file is going to be generated any way on merge, let's do it:

/ocabot merge

**Pull request merged**